Makefile

```diff
@@ -543,6 +544,18 @@ release-publish: release-prereqs
 	@echo "  make VERSION=$(VERSION) release-publish-latest"
 	@echo ""
 
+release-github: hack/bin/hub release-notes
+	@echo "Creating github release for $(VERSION)"
+	hack/bin/hub release create --draft --file $(VERSION)-release-notes.md $(VERSION)
+
+HUB_VERSION?=2.14.2
+hack/bin/hub:
```

Comment:
Wouldn't it make more sense, if we're adding a dependency, to use the (supported) GitHub cli rather than a third-party tool?

Reply:
i just realised hub is now a third-party tool; it definely wasn't before.